recipes/boltons/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set version = "16.3.0" %}
+
+package:
+    name: boltons
+    version: '{{ version }}'
+
+source:
+    git_url: https://github.com/mahmoud/boltons.git
+    git_rev: '{{ version }}'
```

Comment:
Sure. I want the CI to run for 16.3.0 first. But if it builds and passes, I'll make that one final change. 

> On May 23, 2016, at 2:48 PM, Filipe <notifications@github.com> wrote:
> 
> In recipes/boltons/meta.yaml:
> 
> > @@ -0,0 +1,38 @@
> > +{% set version = "16.3.0" %}
> > +
> > +package:
> > +    name: boltons
> > +    version: '{{ version }}'
> > +
> > +source:
> > +    git_url: https://github.com/mahmoud/boltons.git
> > +    git_rev: '{{ version }}'
> Can you change this to:
> 
> fn: boltons-{{ version }}.tar.gz
> url: https://github.com/mahmoud/boltons/releases/download/{{ version }}/boltons-{{ version }}.tar.gz
> sha256: be9aa2b6ffc74b526b1fe080272fd5706e05747f40166e3ca60ea04e34ca8816
> We try to avoid cloning the whole repository, and the checksum is to ensure we have the right file.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>